cmd/res/provisionwatchers/generic.provision.watcher.yaml

```diff
@@ -1,7 +1,7 @@
 name: "USB-Camera-Provision-Watcher"
 serviceName: "device-usb-camera"
 identifiers:
-    Path: "."
+    Paths: "."
```

Comment:
accidental rename I think

Reply:
I renamed the path device field, so I renamed it so the device service sdk could find a match to the Identifiers. I was experiencing errors in discovery, and this seemed to be the fix